cypress/support/fragments/inventory/inventoryInstance.js

```diff
@@ -903,16 +920,19 @@ export default {
     cy.expect(MultiColumnListCell({ content }).exists());
   },
 
-  checkHoldingsTableContent({ name, records = [] } = {}) {
+  checkHoldingsTableContent({ name, records = [], columnIndex = 0, shouldOpen = true } = {}) {
     const holdingsSection = Accordion({ label: including(`Holdings: ${name}`) });
-    cy.do(holdingsSection.clickHeader());
+
+    if (shouldOpen) {
+      cy.do(holdingsSection.clickHeader());
+    }
 
     records.forEach((record, index) => {
       if (record.barcode) {
         cy.expect(
           holdingsSection
             .find(MultiColumnListRow({ rowIndexInParent: `row-${index}` }))
-            .find(MultiColumnListCell({ columnIndex: 0 }))
+            .find(MultiColumnListCell({ columnIndex }))
```

Comment:
Why you delete index?

Reply:
Right now it's passed as parameter to the function, it done for cases when 'move item' operation is active and barcode column shifted 2 columns right

Reply:
thanks